kolibri/plugins/coach/assets/src/composables/useQuizCreation.js

```diff
@@ -477,8 +548,13 @@ export default function useQuizCreation(DEBUG = false) {
   return {
     // Methods
     saveQuiz,
+    initializeWorkingResourcePool,
```

Comment:
This is turning the `useQuizCreation` composable into a bit of a catch all again - I think all the state and methods here are at a smaller level of granularity than the useQuizCreation composable is handling.

I think with some judicious work, this could all be instantiated entirely within the ResourceSelection component, or in a separate composable that is only used inside the ResourceSelection component.

Reply:
I think the main issue is handling that the component re-setups each navigation, hence it living in then composable. I think I initially suggested that it live in ResourceSelection but foresaw it being a problem until we fix the coach routing issues